api/v1beta1/ovncontroller_types.go

```diff
@@ -70,7 +70,13 @@ type OVNControllerSpec struct {
 	// +kubebuilder:validation:Optional
 	// NetworkAttachment is a NetworkAttachment resource name to expose the service to the given network.
 	// If specified the IP address of this network is used as the OvnEncapIP.
+	// Deprecated: superseded by NetworkAttachments
 	NetworkAttachment string `json:"networkAttachment"`
+
+	// +kubebuilder:validation:Optional
+	// NetworkAttachment is a NetworkAttachment resource name to expose the service to the given network.
+	// If specified the IP address of this network is used as the OvnEncapIP.
```

Comment:
is it true? I assume we'll use one of them for encaps only, no?

Reply:
Misleading at best - reworded along with the rest of the comment.